src/quo/components/settings/category/settings/view.cljs

```diff
@@ -7,8 +7,8 @@
     [react-native.core :as rn]))
 
 (defn- category-internal
-  [{:keys [label data] :as props}]
-  [rn/view {:style (style/container label)}
+  [{:keys [label data style] :as props}]
+  [rn/view {:style (merge (style/container label) style)}
```

Comment:
it's preferable to use `container-style` prop as we do this for all other quo components 👍

Reply:
ah that is probably more confusing. If it's not too much effort I would suggest a refactor of the props.
`style`-> `container-style`
`container-style` -> `inner-container-style` or `content-container-style` etc